pages/library/_viewId.vue

```diff
@@ -48,10 +48,10 @@ export default Vue.extend({
       sortBy: 'SortName',
       hasFilters: false,
       isDefaultView: true, // Movie view, not Collection. Music view, not Genres...
-      statusFilter: [],
-      genresFilter: [],
-      yearsFilter: [],
-      ratingsFilter: [],
+      statusFilter: [] as unknown,
```

Comment:
Same here. Isn't it an array of strings?

Reply:
@Camc314 I was not sure but all I checked was indeed a string.

If you're sure it's always a string then I can change it :).

Reply:
Yep, just double checked, it is an array of strings